test/status-repository-dir/expected-stdout

```diff
@@ -1,2 +1,3 @@
 Repository directory:  %s%efoo
 Working directory:     bar/baz
+Manifest path:         .keboola/repository.json
```

Comment:
Info o repository dir, vypisujeme aj cestu ku manifestu (ako je to aj pre project/template dir).